tests/php/UtilCachedContentUrlTest.php

```
@@ -0,0 +1,124 @@
+
+		$first = Util::cached_content_url( '/themes/a/' );
+
+		$blog_id = 2;
+		$second  = Util::cached_content_url( '/themes/a/' );
+		$this->assertNotSame( $first, $second );
+		$this->assertSame( 2, $calls );
+
+		$blog_id = 1;
+		$this->assertSame( $first, Util::cached_content_url( '/themes/a/' ) );
+		$this->assertSame( 2, $calls );
+	}
```

PHPMD (2.15.0)

[error] The method test_results_are_keyed_per_blog_id is not named in camelCase. (undefined)

(CamelCaseMethodName)